src/components/ZoomHandler.component.jsx

```diff
@@ -30,10 +30,14 @@ const ZoomHandler = React.createClass({
 		}
 	},
 	onZoomEnd() {
-		this.props.setZoom(event.transform);
+		if (!isNaN(event.transform.k) && !isNaN(event.transform.x) && !isNaN(event.transform.y)) {
```

Comment:
event here is global (from import)
I don't understand the need to check if k, x & y are numbers.
It looks like pre assertion, should be the setZoom which is doing this check;